tests/integration/test_subscribe.py

```diff
@@ -142,7 +145,10 @@ def test_defer_vcs(self):
 
         dat = self.waitForSub('defer', root)[0]
         self.assertEqual(True, dat['is_fresh_instance'])
-        self.assertEqual([{'name': '.hg', 'exists': True}], dat['files'])
+        dot_hg = '.hg'
```

Comment:
This should no longer be necessary once https://github.com/facebook/watchman/wiki/Better-Unicode-handling-plan is active.